script.js

```diff
@@ -0,0 +1,30 @@
+var hamburgerButton = document.querySelector(".hamburger");
+
+var hamburgerMenu = document.querySelector(".hamburger-menu");
+
```

Comment:
- [ ] great job. Please kindly use the Figma background color `#6070FF` so it can follow the exact design
- [ ] please take the link up a little and also the close icon so it matches the original design.
yours
![Screenshot (862)_LI](https://user-images.githubusercontent.com/65298048/223149917-3d0c24e3-cbf7-4068-9ea0-2e2cb141f13e.jpg)
figma
![Screenshot (862)](https://user-images.githubusercontent.com/65298048/223150164-b57fbcb5-a45b-4ac5-a5fb-fb0079843a28.png)